server/handlers.go

```diff
@@ -174,6 +174,22 @@ func (s *Server) handleAuthorization(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if authReq.Connector != "" {
+		filtered := connectors[:0]
```

Comment:
You should not create slice that way. When you will modify filtered, you will modify also connectors ;)